src/helpers.js

```diff
@@ -65,6 +66,34 @@ export default {
       allureReporter.addAttachment('Execution video', videoPath, 'video/mp4');
     }
 
+    const frames = glob.sync(`${this.recordingPath}/*.png`);
+
+    if (frames.length) {
+      const frameRegex = /^.*\/(\d{4})\.png$/;
+      const frameNumbers = frames.map((path) => +path.replace(frameRegex, '$1'));
+      const pad = (frameNumber) => frameNumber.toString().padStart(4, '0');
```

Comment:
Where does the `4` come from? Can we infer it from a variable?

Reply:
4 here is the zero padding size for the screenshot files - effectively set at https://github.com/webdriverio-community/wdio-video-reporter/blob/949d27f44ab5c44fccd59fac3b211e16c249576f/src/index.js#L266

Reply:
Can we create a descriptive constant and ensure we use it everywhere it is needed?